tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -1680,10 +1680,25 @@ bool CUDABlas::DoBlasGemm(
   return DoBlasInternal(
       dynload::cublasSgemmEx, stream, true /* = pointer_mode_host */,
       CUDABlasTranspose(transa), CUDABlasTranspose(transb), m, n, k, &alpha,
-      CUDAMemory(a), CUBLAS_DATA_HALF, lda,
-      CUDAMemory(b), CUBLAS_DATA_HALF, ldb,
-      &beta,
-      CUDAMemoryMutable(c), CUBLAS_DATA_HALF, ldc);
+      CUDAMemory(a),
```

Comment:
I wonder if it makes sense defining our own "half type" somewhere in another header that we typedef once, so we centralize the definition of the symbol.  Thoughts @zheng-xq  ?